tests/models/siglip/test_modeling_siglip.py

```diff
@@ -55,6 +67,178 @@
     from transformers import SiglipProcessor
 
 
+class SiglipModelTesterMixin(ModelTesterMixin):
```

Comment:
cc @ydshieh for comments / opinions on this mixin structure within the model's testing file

Reply:
IIRC, it's just to overwrite `test_eager_matches_sdpa_inference` (which is a large block) in `ModelTesterMixin)`.

So we don't really need to have this new class. However, there are 3 or more model test classes in this file. It's nice to have `SiglipModelTesterMixin` and just overwrite with mini block like

```
    def test_eager_matches_sdpa_inference(self, torch_dtype: str):
        super().test_eager_matches_sdpa_inference(
            torch_dtype=torch_dtype,
            logit_keys=("pooler_output", "last_hidden_state"),
            use_attnetion_mask_options=(False,),
        )
```